app/root.css

```diff
@@ -378,10 +375,16 @@ footer > *:not(:last-child) {
   width: 100%;
 }
 
-.dropdown button:hover {
+.result-item:hover {
   background-color: #ddd;
 }
 
+.result-item.already-open {
+  color: gray;
+  cursor: default;
+  background-color: transparent;
```

Comment:
@ccstan99 I added a way to style search results for questions that are already open - this way:
* it won't go away during multiselect (Shift+click)
* we don't have to return more than 5 results from the worker for filtering
* clicking on grayed search result still brings it to top of page as an escape hatch for people who would gets lost and used the search feature to find what is already "somewhere" on the page

Feel free to customize the styles and/or create follow-up GH issues for better functionality (e.g. different color for all previously visited questions in this browser, not just the currently open ones).

<img width="473" alt="already open" src="https://user-images.githubusercontent.com/1087670/177530496-95eba724-75c4-4989-9eb9-f9ef6d7f81b4.png">